lib/node_modules/@stdlib/math/base/special/normhermite-polynomial/README.md

```diff
@@ -28,10 +28,10 @@ limitations under the License.
 
 The normalized (aka "probabilist") [Hermite polynomials][hermite-polynomial] are given by
 
-<!-- <equation class="equation" label="eq:normalized_hermite_polynomials" align="center" raw="TODO" alt="Equation for normalized Hermite polynomials."> -->
+<!-- <equation class="equation" label="eq:normalized_hermite_polynomials" align="center" raw="He_{n}(x)=(-1)^{n} e^{\frac{x^2}{2}} \frac{\mathrm d^{n}}{\mathrm d x^{n}} e^{-\frac{x^2}{2}}" alt="Equation for normalized Hermite polynomials."> -->
 
 <div class="equation" align="center" data-raw-text="He_{n}(x)=(-1)^{n} e^{\frac{x^2}{2}} \frac{\mathrm{d}^{n}}{\mathrm{d}x^n} e^{-\frac{x^2}{2}}" data-equation="eq:normalized_hermite_polynomials">
-    <img src="https://cdn.rawgit.com/jrreed83/stdlib/7872ed2882687705fe000f5ee333b254506d5bc0/lib/node_modules/@stdlib/math/base/special/normhermite-polynomial/docs/img/equation_normalized_hermite_polynomials.svg" alt="Equation for normalized Hermite polynomials.">
+    <img src="https://cdn.rawgit.com/jrreed83/stdlib/3016d0a4faf9ef8b647c384b60b659bb1acd99cf/lib/node_modules/@stdlib/math/base/special/normhermite-polynomial/docs/img/equation_normalized_hermite_polynomials.svg" alt="Equation for normalized Hermite polynomials.">
```

Comment:
What follows is mostly a dialogue to myself. Turns out we do need to do this (setting the equation URL) on our end. Not much you can do about this, unfortunately. The issue is that the URL points to your repo, which we cannot guarantee will always be around. Were you to delete your forked repo, the URL would 404. Nothing for you to do now wrt this. I will update upon merging.

Reply:
Okay.  I should have caught that issue in the URL.